src/applications/personalization/profile/components/notification-settings/NotificationSettings.jsx

```diff
@@ -27,17 +31,48 @@ import HealthCareGroupSupportingText from './HealthCareGroupSupportingText';
 import MissingContactInfoAlert from './MissingContactInfoAlert';
 import NotificationGroup from './NotificationGroup';
 
+const SelectNotificationOptionsAlert = ({ firstChannelId }) => {
+  return (
+    <div data-testid="select-options-alert">
+      <AlertBox
+        status={ALERT_TYPE.WARNING}
+        headline="Select your notification options"
+        level={2}
+      >
+        <div>
+          <p>
+            We’ve added notification options to your profile. Tell us how you’d
+            like us to contact you.
+          </p>
+          <p>
+            <a
+              href={`#${firstChannelId}`}
+              // className="vads-u-text-decoration--none vads-u-padding--1 vads-u-margin-left--neg1 vads-u-margin-top--neg1 vads-u-margin-bottom--neg1"
```

Comment:
Can these classes just be removed or are they needed later?

Reply:
oops, thanks for catching that. I actually do want those and had just commented them out for a quick test.